doc/source/download.rst

```diff
@@ -291,6 +296,31 @@ You can download and install gdal using the vcpkg dependency manager:
 
 If the version is out of date, please `create an issue or pull request <https://github.com/Microsoft/vcpkg>`__ on the vcpkg repository.
 
+Spack
+................................................................................
+
+Spack is a package management tool designed to support multiple versions and
+configurations of software on a wide variety of platforms and environments.
+It was designed for large supercomputing centers. Spack builds packages from
+sources, and allows tweaking their configurations.
+
+You can find information about GDAL in Spack at
+https://spack.readthedocs.io/en/latest/package_list.html#gdal
+
+For the default GDAL build with a reduced number of drivers:
+
+::
+
+    git clone -c feature.manyFiles=true https://github.com/spack/spack.git
+    cd spack/bin
+    ./spack install gdal
+
+For a build with netcdf driver enabled (cf https://github.com/spack/spack/issues/12586 for the explanation of the ^mpich~fortran workaround)
+
+::
+
+    ./spack install gdal +netcdf ^mpich~fortran
```

Comment:
Is this workaround actually needed? The linked issue seemed inconclusive, but is from 3 years ago and was likely fixed by HDF5 since then.

Reply:
> Is this workaround actually needed? The linked issue seemed inconclusive, but is from 3 years ago and was likely fixed by HDF5 since then.

This was necessary for me on Ubuntu 20.04

Reply:
Can you open a new issue or comment on the old issue? I would rather fix this than document a workaround.

Reply:
==> https://github.com/spack/spack/issues/33857